doc/source/examples/rpt/particle-fem-reconstruction/particle-fem-reconstruction.rst

```diff
@@ -0,0 +1,162 @@
+- Solver: ``rpt_fem_reconstruction_3d``
+- Displays the use of the inverse FEM in reconstruction of the radioactive particle positions
+
+Files used in this example
+---------------------------
+- Parameter file: ``examples/rpt/particle-fem-reconstruction/rpt-fem-reconstruction.prm``
+- File containing detectors positions: ``examples/rpt/particle-fem-reconstruction/positions.detector``
+- File containing the experimental counts: ``examples/rpt/particle-fem-reconstruction/experimental-counts.txt``
+- File containing the real particle positions: ``examples/rpt/particle-fem-reconstruction/real-positions.txt``
+
+Description of the case
+-------------------------
+The purpose of the Finite Element Method based Position Reconstruction (FEM-PR) is to filter the noise resulting from the Monte Carlo model by using an
+orthogonal projection of the Monte Carlo model onto the domain. Then, the algorithm searches through the cells and it
+minimizes the least-squares error between the photon count obtained from the FEM interpolation and the experimental count for all the detectors in each cell.
+The 3D domain is discretized using linear tetrahedral elements and an approximation of the physical values inside an element can be written in terms of shape functions and natural
+coordinates as:
+
+
+
+.. math::
+    \varphi_{x,y,z}=\Sigma^{n}_{i=1}N_{i\varphi_i}
+
+where :math:`\varphi` is a continuous physical value, :math:`\varphi_{x,y,z}` is the value at the global coordinate point :math:`(x,y,z)`, :math:`\varphi_i` are the values at the nodes of the element and :math:`n` is the number of nodes of the element (:math:`n=4` for a linear tetrahedral element), and :math:`N_i` are the shape functions defined as follows :
+
+.. math::
+    N_1=1-\eta-\xi-\zeta,\,\,\, N_2=\eta, \,\,\, N_3=\xi, \,\,\, N_4= \zeta
+
+
+Therefore, the isoparametric approximation combined with a search algorithm can calculate the tracer’s position using a set of photon
+counts received at each detector. The search algorithm goes through each cell inside the domain and solves a
+least-squares cost-function defined as:
+
+.. math::
+    F=\sum_{i=1}^{N_D} (C^{i}_{num}-C^{i}_{exp})^2
+
+where :math:`C_{num}` and :math:`C_{exp}` are the numerical and experimental counts, respectively, :math:`N_{D}` is the number of detectors, and :math:`i` is the number associated with the detector. The main search hypothesis is that the tracer is located inside the search element.
+
+
+This example shows how using the tuned parameters of each detector we generate the dictionary for the corresponding geometry
+(shown in the illustration below) and then we reconstruct the positions of radioactive particles based on the received photon counts by the detectors.
+
+
+.. image::  images/syst-from-above.png
+    :alt: The geometry
+    :align: center
+    :name: geometry_description
+    :width: 380
+
+
+
+Parameter file
+----------------
+
+*rpt-fem-reconstruction.prm* file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Remove these lines since this subtitle is only used whenever there are more than one parameter files:
```suggestion
```